docs/architecture/microservices/net-core-net-framework-containers/net-container-os-targets.md

```diff
@@ -29,6 +32,11 @@ When you add the image name to your Dockerfile file, you can select the operatin
 | mcr.microsoft.com/dotnet/core/aspnet:2.2-alpine | .NET Core 2.2 runtime-only on Linux Alpine distro |
 | mcr.microsoft.com/dotnet/core/aspnet:2.2-nanoserver-1803 | .NET Core 2.2 runtime-only on Windows Nano Server (Windows Server version 1803) |
 
+## Additional resources
+
+- **Server Core for .Net Framework Insufficient (GitHub issue)**  
+  <https://github.com/dotnet/docs/issues/13409>
```

Comment:
why list the docs issue as an additional resource?

Reply:
The real issue seems to be discussed at https://github.com/microsoft/dotnet-framework-docker/issues/299. Wouldn't that be a better place to link to?

Reply:
Yep!